src/game/cards/events/rain.ts

```diff
@@ -37,11 +37,16 @@ export const rain: IEvent = Object.freeze<IEvent>({
         }
       })
 
-      // TODO: Show a notification indicating that all crops were watered
-
       match = updateField(match, playerId, { crops })
     }
 
+    context.shell.triggerNotification({
+      type: ShellNotificationType.GENERIC_INFO,
+      payload: {
+        message: 'Watered all crops',
+      },
+    })
```

Comment:
Move the definition of the notification content to `src/ui/components/Match/useSnackbar.ts` to align to the established pattern of centralized notification content management.